src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -50,22 +54,20 @@ public void armSpeed(double speed) {
     public boolean isInBound(double speed){
         if((getPosition() < 3) && (speed < 0)) return false;
         if ((getPosition() > 113) && (speed > 0)) return false;
-        
         return true;
     }
 
     public double getPosition(){
```

Comment:
What units does this method return?  The encoder returns rotations, when you convert to degrees, which you divide by.... what to convert into what units?

Reply:
The 16.0 needs to go into constants as ARM_GEAR_RATIO.  getPosition () can return a Rotation2d object if desired, or the method should be renamed getPositionDeg().

Reply:
I also thought Rotation2d was a better idea, the code has been changed to return Rotation2d for both getVelocity() and getPosition().

The gear ratio is in Constants.java ✅